akka-actor-typed-tests/src/test/scala/akka/actor/typed/MailboxSelectorSpec.scala

```diff
@@ -64,21 +64,22 @@ class MailboxSelectorSpec extends ScalaTestWithActorTestKit("""
 
     "set capacity on a bounded mailbox" in {
       val latch = new CountDownLatch(1)
+      val probe = testKit.createTestProbe[String]()
       val actor = spawn(Behaviors.receiveMessage[String] {
         case "one" =>
           // block here so we can fill mailbox up
+          probe ! "blocking-on-one"
           latch.await(10, TimeUnit.SECONDS)
           Behaviors.same
         case _ =>
           Behaviors.same
       }, MailboxSelector.bounded(2))
       actor ! "one" // actor will block here
+      probe.expectMessage("blocking-on-one")
       actor ! "two"
-      LoggingTestKit.deadLetters().withCheckExcess(false).expect {
-        // one or both of these doesn't fit in mailbox
-        // depending on race with how fast actor consumes
```

Comment:
I can't understand that comment. two and three will alway be in the mailbox, waiting for one. Then four will exceed the capacity.

Reply:
I think since we didn't wait for "one" to actually be processed, it could still be in the mailbox when we enqueued "two" and "three".

Reply:
ah, of course